### PR TITLE
PMM 3874 Add standard labels to QAN API

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-3872

### CODE REVIEW

File: api/qanpb/collector.proto [hunk 1]
```diff
@@ -62,6 +62,26 @@ message MetricsBucket {
...
+  // Environment name
+  string environment = 161;
+  // Availability zone
+  string az = 162;
+  // Region name
+  string region = 163;
+  // Node model
+  string node_model = 164;
+  // Container name
+  string container_name = 165;
```
Comment (AlekSi) [inline api/qanpb/collector.proto]: I think we should move other standard labels like `agent_uuid` (please rename it to `agent_id`) and `metrics_source` (please rename it to `agent_type`) to that group, and separate it from other fields like `period_start_unix_secs` below.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-336/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-336/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-336/

**Pull request merged**